schedule/kernel/agama_install_ltp.yaml

```diff
@@ -0,0 +1,20 @@
+name: agama_install_ltp
+description:    >
+    Universal installation schedule for Agama to prepare bare metal using IPXE or
+    prepare PowerVM and install LTP
+schedule:
+    - '{{boot}}'
+    - installation/agama_reboot
+    - installation/grub_test
+    - installation/first_boot
+    - console/system_prepare
+    - kernel/install_ltp
+    - kernel/shutdown_ltp
+conditional_schedule:
+    boot:
+        IPXE:
+            1:
+                - installation/bootloader_start
+        BACKEND:
+            spvm:
```

Comment:
One last bit.  It needs to be `pvm_hmc`, sle16 doesn't run on `spvm`, because it is Power8 only.